src/jasper_helpers/links.cr

```diff
@@ -26,4 +26,14 @@ module JasperHelpers::Links
       content(:button, body, {:type => "submit"})
     end
   end
+
+  def button_to(body : String, url : String, method = :post)
+    button_content = yield
+    form(action: url, class: "button", method: method) do
+      String.build do |str|
+        str << button_content
```

Comment:
Since you're not doing any checks on the yielded result first it makes more sense to yield directly into the string builder.
`str << yield` This saves time and memory and IMO is more readable.